wallet/thread/photos.go

```diff
@@ -124,6 +126,17 @@ func (t *Thread) HandleDataBlock(message *pb.Envelope, signed *pb.SignedThreadBl
 		}
 	}
 
+	// pin data
```

Comment:
Pin the "off-chain" data... when we get to comments, will need a data type param on the message so we can determine what the dir structure should be, i.e., are we pinning a photo, comment, something else, etc.